src/plone/restapi/serializer/portlet.py

```diff
@@ -335,6 +372,30 @@ def get_view_url(context):
     return name, item_url
 
 
+class NewsPortletRenderer(NewsRenderer):
+    def render(self):
+        items = []
+        news = self.published_news_items()
+        for new in news:
+            itemList = getMultiAdapter((new, self.request), ISerializeToJsonSummary)()
+            itemList['icon'] = new.getObject().getIcon()
+            items.append(itemList)
+        res = {'items': items}
+        return res
+
+
+class StaticPortletRenderer(StaticRenderer):
+    def render(self):
```

Comment:
I think "data" comes from Assignment, this information is serialized already, only need to include text.